tests/test-webapps/test-owb-cdi-webapp/src/main/webapp/WEB-INF/jetty-env.xml

```diff
@@ -1,9 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
 
 <Configure id="webAppCtx" class="org.eclipse.jetty.webapp.WebAppContext">
+  <Call name="setInitParameter">
+    <Arg>openwebbeans.web.sci.active</Arg>
```

Comment:
Maybe a comment here or link to documentation, so it is clear that this is important.

Reply:
Fixed.